app/helpers/containers_helper.rb

```diff
@@ -32,18 +32,18 @@ def container_link_hash(container, resource)
   def container_title_actions(container)
     @compute_resource = container.compute_resource
     title_actions(
-        button_group(
-          link_to(_('Commit'), '#commit-modal', :'data-toggle' => 'modal')
-        ),
-        button_group(container_power_action(container.in_fog)),
-        button_group(
-          display_delete_if_authorized(
-            hash_for_container_path(:id => container.id)
-                                    .merge(:auth_object => container,
-                                           :auth_action => 'destroy',
-                                           :authorizer  => authorizer),
-            :confirm     => _("Delete %s?") % container.name)
-        )
+      button_group(
+        link_to(_('Commit'), '#commit-modal', :'data-toggle' => 'modal')
+      ),
+      button_group(container_power_action(container.in_fog)),
+      button_group(
+        display_delete_if_authorized(
+          hash_for_container_path(:id => container.id)
+          .merge(:auth_object => container,
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use the new Ruby 1.9 hash syntax.